src/main/java/org/openrewrite/java/spring/boot2/MigrateDatabaseCredentials.java

```diff
@@ -83,7 +83,7 @@ public TreeVisitor<?, ExecutionContext> getVisitor() {
                 public Yaml visitDocuments(Yaml.Documents documents, ExecutionContext ctx) {
                     if (FindProperty.find(documents, "spring." + tool + ".username", true).isEmpty() &&
                             FindProperty.find(documents, "spring." + tool + ".password", true).isEmpty()) {
-                        doAfterVisit(new FindProperty("spring." + tool + ".url", true).getVisitor());
+                        doAfterVisit(new FindProperty("spring." + tool + ".url", true, null).getVisitor());
```

Comment:
This change was needed to make it compile again. The `FindProperty` classes has just been changed by: 
- https://github.com/openrewrite/rewrite/pull/4657

Reply:
That type of change may be pushed to main directly. No need to do that now, but just so you know.